src/app/snapshot/models/filters.model.ts

```diff
@@ -3,6 +3,7 @@ import { SnapshotType } from '../../shared/models';
 export interface Filters {
   accounts: string[];
   vmIds: string[];
+  volumeVmIds: string[];
```

Comment:
volumeVmIds is not entirely clear. Rename or add comment to describe what volumeVm means